BoostTestPlugin/source.extension.vsixmanifest

```diff
@@ -1,39 +1,39 @@
+        <DisplayName>Test Adapter for Boost.Test</DisplayName>
+        <Description xml:space="preserve">Enables Visual Studio's testing tools with unit tests written for Boost.Test.</Description>
+        <PackageId>Microsoft.VisualStudio.VC.Ide.TestAdapterForBoostTest</PackageId>
+        <MoreInfo>https://go.microsoft.com/fwlink/?linkid=853712</MoreInfo>
+        <License>use.txt</License>
+        <GettingStartedGuide>https://go.microsoft.com/fwlink/?linkid=853711</GettingStartedGuide>
+        <ReleaseNotes>Release Notes.txt</ReleaseNotes>
+        <Icon>Icons\Icon.ico</Icon>
+        <PreviewImage>Boost_preview.png</PreviewImage>
+        <Tags>C++, Unit Testing, Testing, test explorer, TDD, Boost, Boost Test Library, Visual Studio, Test Adapter</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Pro" />
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Enterprise" />
+    </Installation>
+    <Dependencies>
+        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,]" />
+        <Dependency d:Source="Installed" d:ProjectName="BoostTestAdapter" Id="Microsoft.VisualStudio.MPF.15.0" DisplayName="Visual Studio MPF 15.0" Version="[15.0,17.0)" />
+    </Dependencies>
+    <Assets>
+        <Asset d:Source="Project" d:ProjectName="BoostTestAdapter" Type="Microsoft.VisualStudio.MefComponent" Path="|BoostTestAdapter|" />
+        <Asset d:Source="Project" d:ProjectName="BoostTestAdapter" Type="UnitTestExtension" Path="|BoostTestAdapter|" />
+        <Asset Type="Application Settings Configuration file" d:Source="File" Path="BoostTestAdapter.dll.config" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="VisualStudioAdapter" Path="|VisualStudioAdapter|" AssemblyName="|VisualStudioAdapter;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="Antlr.DOT" Path="|Antlr.DOT|" AssemblyName="|Antlr.DOT;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="BoostTestShared" Path="|BoostTestShared|" AssemblyName="|BoostTestShared;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="BoostTestPackage" Path="|BoostTestPackage;PkgdefProjectOutputGroup|" />
+        <Asset Type="Microsoft.VisualStudio.ItemTemplate" d:Source="Project" d:ProjectName="BoostTestItemTemplate" d:TargetPath="|BoostTestItemTemplate;TemplateProjectOutputGroup|" Path="ItemTemplates" d:VsixSubPath="ItemTemplates" />
+    </Assets>
+    <Prerequisites>
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,)" DisplayName="Visual Studio core editor" />

```

Comment:
These are also 15.8. I don't know if they should be 17.0 or not.

Reply:
Same as above, should be safe and seemed ok in testing.